content/release-notes/2022-07-19.md

```diff
@@ -18,16 +18,16 @@ label: 'Storage'
   - `wal_acceptors` to `safekeepers`
 - Control Plane: Renamed `zenith_admin` role to `cloud_admin`.
 - Pageserver: Implemented a page service `fullbackup` endpoint that works like basebackup but also sends relational files.
-- Pageserver: Added support for importing a basebackup taken from vanilla PostgreSQL or another pageserver via psql copy in protocol.
-- Pageserver: Fixed the database size calculation to count VMs and FSMs in addition to the main fork of the relation.
-- Pageserver: Updated the timeline size reported when DROP DATABASE is executed.
+- Pageserver: Added support for importing a base backup taken from a standalone PostgreSQL instance or another Pageserver using `psql` copy.
+- Pageserver: Fixed the database size calculation to count Visibility Maps (VMs) and Free Space Maps (FSMs) in addition to the main fork of the relation.
```

Comment:
Not sure that Postgres docs mentions them as plurals https://www.postgresql.org/docs/current/storage-vm.html

Reply:
I did check that. They do mention "VMs", so I think it's OK.